mqtt/mqttd/src/broker/bootstrap/edgehub.rs

```diff
@@ -189,15 +197,11 @@ async fn start_sidecars(
         if let Err(e) = command_handler_join_handle.await {
             error!(message = "failed waiting for command handler shutdown", error = %e);
         }
-        if let Err(e) = bridge.await {
-            error!(message = "failed waiting for bridge shutdown", error = %e);
-        }
```

Comment:
This was not correct either so I moved it. Since it is not fail-able and doesn't depend on the broker I put it in the start_sidecars() method. It doesn't actually do anything yet though.

When it is implemented it will be structured the same way as the command handler. Ideally with fail-able init steps, then in a separate thread the sidecars will run and get shutdown.